specification/templates/CONTRIBUTING.md

```diff
@@ -75,6 +75,6 @@ any 'help wanted' issues is a great place to start.
 See the [LICENSE](LICENSE) file for our repository's licensing. We will ask you to
 confirm the licensing of your contribution.
 
-We may ask you to sign a [Contributor License Agreement
-(CLA)](http://en.wikipedia.org/wiki/Contributor_License_Agreement) for larger
-changes.
+## CLA
+
+All contributors must sign the [Splunk Contributor License Agreement (CLA)](https://www.splunk.com/en_us/form/contributions.html).
```

Comment:
I just checked and the whole file is written in "2nd form".
Therefore it would be better to rephrase it to:

```suggestion
Before contributing, you must sign the [Splunk Contributor License Agreement (CLA)](https://www.splunk.com/en_us/form/contributions.html).
```

Also maybe it would be better just to put it as a bullet in the `Contributing via Pull Requests` section?